### PR TITLE
Small edits, code formatting and typos

### PR DESCRIPTION
**Pull request by chStaiger**
Before moving to the new location, I went through the RTD documents again and made some small edits. No major changes.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@ irods-basicGUI/logs/*
 irods-iBridgesGui.spec
 *DS_Store
 docs/source/api/generated
+docs/outpudir
```
Comment (qubixes) [inline .gitignore:7]: Why is this there? Also, spelling error?

File: docs/source/metadata.rst [hunk 2]
```diff
@@ -67,7 +67,7 @@ Below are examples on how to delete metadata entries:
 Export Metadata
 ---------------
 
-The method :meth:`MetaData.to_dict` will provide you with a python dictionary containing the user-defined metadata:
+The method :meth:`meta.to_dict` will provide you with a python dictionary containing the user-defined metadata:
```
Comment (qubixes) [inline docs/source/metadata.rst:70]: This doesn't work, now the link is broken. What was wrong with the original?

File: docs/source/quickstart.rst [hunk 3]
```diff
@@ -131,7 +131,7 @@ Let's add some metadata to a collection or data object:
     ipath = IrodsPath(session, "/irods/path")
     ipath.meta.add("some_key", "some_value", "some_units")
 
-We have used the :code:`IrodsPath` class here, which is central to the iBridges API. From here we have
+We have used the :code:`IrodsPath` class here, which another central class to the iBridges API. From here we have
```
Comment (qubixes) [inline docs/source/quickstart.rst:134]:
```suggestion
We have used the :code:`IrodsPath` class here, which is another central class to the iBridges API. From here we have
```

**Review by qubixes — CHANGES_REQUESTED** [review]
Thanks for finding the small issues in our documentation. Just have a few comments, as usual...

File: .gitignore (hunk 1 above)
Comment (chStaiger) [reply .gitignore:7]: Indeed ... spelling error... should be outputdir

Comment (qubixes) [reply .gitignore]: How does it get generated? I have never seen it come up spontaneously?

**Pull request merged**